src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/WebProfilerExtension.php

```diff
@@ -54,23 +54,13 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('web_profiler.debug_toolbar.mode', $config['toolbar'] ? WebDebugToolbarListener::ENABLED : WebDebugToolbarListener::DISABLED);
         }
 
-        $baseDir = array();
-        $rootDir = $container->getParameter('kernel.root_dir');
-        $rootDir = explode(DIRECTORY_SEPARATOR, realpath($rootDir) ?: $rootDir);
-        $bundleDir = explode(DIRECTORY_SEPARATOR, __DIR__);
-        for ($i = 0; isset($rootDir[$i], $bundleDir[$i]); ++$i) {
```

Comment:
this was just an algorithm looking for the project_dir based on what were available before the parameter existed